lib/babe/build_integration_test.go

```diff
@@ -54,41 +54,10 @@ func TestSeal(t *testing.T) {
 	require.True(t, ok, "could not verify seal")
 }
 
-func createTestBlock(t *testing.T, babeService *Service, parent *types.Header,
-	exts [][]byte, slotNumber, epoch uint64, epochData *epochData) *types.Block {
-	for _, ext := range exts {
-		vtx := transaction.NewValidTransaction(ext, &transaction.Validity{})
-		_, err := babeService.transactionState.Push(vtx)
-		require.NoError(t, err)
-	}
-
-	duration, err := time.ParseDuration("1s")
-	require.NoError(t, err)
-
-	slot := Slot{
-		start:    time.Now(),
-		duration: duration,
-		number:   slotNumber,
-	}
-
-	bestBlockHash := babeService.blockState.BestBlockHash()
-	rt, err := babeService.blockState.GetRuntime(bestBlockHash)
-	require.NoError(t, err)
-
-	preRuntimeDigest, err := claimSlot(epoch, slotNumber, epochData, babeService.keypair)
-	require.NoError(t, err)
-
-	block, err := babeService.buildBlock(parent, slot, rt, epochData.authorityIndex, preRuntimeDigest)
-	require.NoError(t, err)
-
-	babeService.blockState.(*state.BlockState).StoreRuntime(block.Header.Hash(), rt)
-	return block
-}
-
-// TODO: add test against latest dev runtime
-// See https://github.com/ChainSafe/gossamer/issues/2704
+// TODO see if there can be better assertions on block body
```

Comment:
Is this TODO still pending?

Reply:
Added some of these questions to the comment to provide some research guidance